cli/cli/commands/lsp/lsp.go

```diff
@@ -0,0 +1,15 @@
+package lsp
+
+import (
+	"github.com/kurtosis-tech/kurtosis/lsp"
+	starlark_lsp_cli "github.com/kurtosis-tech/vscode-kurtosis/starlark-lsp/pkg/cli"
+	"github.com/spf13/cobra"
+)
+
+func NewLspCommand() *cobra.Command {
```

Comment:
LSP naming is a very technical name, what reveals implementation details we don't care much for. I wonder if we can change this to something more seamless like `kurtosis dev` or `kurtosis code` (we could even open `code` on `.` for users)

Reply:
Cool, if this is hidden I'm fine with this naming